core/templates/pages/exploration-editor-page/settings-tab/settings-tab.component.spec.ts

```diff
@@ -736,6 +737,32 @@ describe('Settings Tab Component', () => {
       expect(ctrl.isExplorationLockedForEditing()).toBe(false);
     });
 
+    it('should call closeRolesForm in reassignRole', fakeAsync(() => {
```

Comment:
ditto

Reply:
tried to make the test name more clear ( Done )